lib/stripe/api_resource.rb

```diff
@@ -48,6 +48,15 @@ def self.save_nested_resource(name)
       end
     end
 
+    def self.custom_method(name, http_method, path = nil)
+      path ||= name.to_s
```

Comment:
What do you think about just running a basic validation on `http_method` (that it's a verb we recognize) here? It'll make a mistake more obvious as it occurs at load time instead of runtime.